### DISCUSSION TITLE
✅ Fail to start SSH

### DISCUSSION THREAD

**Discussion by ghost** [isontheline/pro.webssh.net#1087] [Q&A] [answered]
Hi All

Sorry, I'm kind of a newb in SSH.
I'm trying to start an SSH connection and it fails with the following error:

<EngineLog>
  <BeginAccepting>
    <EngineVersion>9.5.0.97</EngineVersion>
    <EngineStatus>2</EngineStatus>
    <listenPort>443</listenPort>
    <dynamicPortForwarding>0</dynamicPortForwarding>
    <StaticPortForwarding>
      <destPort>80</destPort>
      <destHostname>443</destHostname>
    </StaticPortForwarding>
    <error>The listen thread already exited (1)</error>
    <listenThreadLog></listenThreadLog>
    <error>Failed to start the background listen thread (2)</error>
    <error>Failed.</error>
  </BeginAccepting>
</EngineLog>

The port forwarding I'm using is:

3306:10.5.244.145:3306,443:10.5.244.145:443,5091:10.5.244.145:5091,5101:10.5.244.145:5101,5102:10.5.244.145:5102,5103:10.5.244.145:5103,80:10.5.244.145:80

I'm using a very similar setup in a different connection, just changing the ports being forwarded now...
Can someone help me understand why this is not working?

**Comment by isontheline** [comment, marked as answer]
Hello @vcleao 👋

You may explode each forwards to put them one by one to find the already opened one.

Most probably ports 443 and 80 couldn't be forwarded on your device as other software may have already opened them.
